### PR TITLE
docs: init privacy policy

### PR DESCRIPTION
**Pull request by nugaon**

### CODE REVIEW

File: PRIVACY_POLICY.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Swarm-Extension Privacy Policy
+
+The Swarm Extension belongs to the Swarm Foundation with the purpose of offering web3 environment
```
Comment (crtahlin) [inline PRIVACY_POLICY.md:3]: belongs? not sure what that implies legally, but maybe change to "was developed"?

File: PRIVACY_POLICY.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+for running decentralised applications (dApps). It means, it provides similar functionalities as web2 browsers,
```
Comment (crtahlin) [inline PRIVACY_POLICY.md:4]: Remove "It means,", superfluous.

File: PRIVACY_POLICY.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+so sensitive information may be handled by the browser-extension such as Bee client configuration
+or cross-domain localStore. Although the extension handles these sensitive information for
+the basic security and functionality of dApps, the Swarm Foundation does not share any of these data
+with other 3rd parties, nor collect it.
```
Comment (crtahlin) [inline PRIVACY_POLICY.md:8]: To make it clearer, maybe reword:

"Although the extension handles these sensitive information for
the basic security and functionality of dApps, the Swarm Foundation does not collect this information and therefore also cannot and does not share it with 3rd parties."

File: PRIVACY_POLICY.md (hunk 2 above)
Comment (crtahlin) [inline PRIVACY_POLICY.md:4]: This whole sentence is unclear to me. It extends the functionality of web2 browsers, so that sensitive information can be handled by the extension,. An example of such sensitive information being Bee client configuration or cross-domain localStore?

File: PRIVACY_POLICY.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+By installing Swarm Extension, you accept this privacy policy.
+
+## Do we collect any information from you?
+
+We do not collect any data from you, including personal data.
```
Comment (crtahlin) [inline PRIVACY_POLICY.md:14]: We --> Swarm foundation ? does not collect any data about you.

File: PRIVACY_POLICY.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+## Sharing Data By The Help of The Extension
+
+In Swarm Extension, you can set the connection configuration of an Ethereum Swarm
+Bee client node (that is a P2P storage client). The dApps can utilize this by calling
```
Comment (crtahlin) [inline PRIVACY_POLICY.md:19]: P2P -->peer-to-peer (P2P)

File: PRIVACY_POLICY.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+with other 3rd parties, nor collect it.
+
+By installing Swarm Extension, you accept this privacy policy.
+
+## Do we collect any information from you?
+
+We do not collect any data from you, including personal data.
+
+## Sharing Data By The Help of The Extension
+
```
Comment (crtahlin) [inline PRIVACY_POLICY.md:17]: The Swarm Extension can be configured to connect to an Ethereum Swarm Bee client node (which is a peer-to-peer (P2P) storage client).

should/can we use the word Ethereum in the text?

File: PRIVACY_POLICY.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+in order to connect dApps to web3 services. This configuration is stored in
+the extension's localstore as well. At swarm-extension usage, some centralised
+web3 service provider request will be redirected to the configured Bee client.
+For example, all `https://*.bzz.link` URLs will be cancelled and will be tunneled
+to your Bee client. The BZZ protocol also affects your default search engine usage
+because if you search anything starting with `bzz://`, then it will redirect to your
+Bee client's bzz path with the given BZZ reference.
+
+The extension also injects a library to every webpage so that dapps can communicate
+with your Bee node and utlize web3 features Because of that, webpages can detect
```
Comment (crtahlin) [inline PRIVACY_POLICY.md:43]: utilize

File: PRIVACY_POLICY.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+such content is the dApps responsibility.
+
+## Under The Hood
+
+DApp developers can store user session data to the user's swarm-extension storage.
+It provides a cross-domain localstorage functionality that dApps can utilize
+in order to save and load the same session data on any gateway host.
+If dApps use the extension's private localstorage, it depends on the dapp in question
+what data will be stored and how, like also in the case of data storing on Swarm network.
+The extension does not share the userdata with other 3rd party, or collect it.
```
Comment (crtahlin) [inline PRIVACY_POLICY.md:31]: Can dApps potentially access each other's data, that is stored in the extension? And can the user control that?

File: PRIVACY_POLICY.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+because if you search anything starting with `bzz://`, then it will redirect to your
+Bee client's bzz path with the given BZZ reference.
+
+The extension also injects a library to every webpage so that dapps can communicate
+with your Bee node and utlize web3 features Because of that, webpages can detect
+you have Swarm Extension installed by null-checking `window.swarm` object.
+
+All before mentioned features along with others in this project aim at the decentralised usage of the internet.
+Because the configured Bee client can be on any host (e.g. `http://localhost:1633` or `https://swarmgateway.org`)
+we need permission to any host in order to handle Ethereum Swarm ecosystem domainless components.
```
Comment (crtahlin) [inline PRIVACY_POLICY.md:48]: we -->the extension needs your permission to connect to any host?

File: PRIVACY_POLICY.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+These FakeURL calls will be redirected to the corresponding configured Bee client endpoints based on the configuration,
+that you provided in the extension's popup page.
+
+## Will The Privacy Policy Be Changed?
+
+As new versions are coming the Privacy Policy may be affected by that.
+Please, check this document every time when a new release come out to be up-to-date
+with the changes, it is your responsibility to do so. You can follow back what changes
+happened by using [history feature of GitHub](https://github.com/ethersphere/swarm-extension/commits/main/PRIVACY-POLICY.md).
+Any time when a new version of this privacy policy is posted, you aggree to all changes that are made.
```
Comment (crtahlin) [inline PRIVACY_POLICY.md:59]: ..., by using the Swarm Extension, you agree to all the changes that were made.

**Review by crtahlin — COMMENTED** [review]
Commented on the text.

File: PRIVACY_POLICY.md (hunk 8 above)
Comment (nugaon) [reply PRIVACY_POLICY.md:31]: depends on what kind of data.
dApp cannot access each other's localStorage, but they upload unencrypted data to Swarm (by the user), then other dApps can read it.

**Review by agazso — APPROVED** [review]
